scripts/download.pl

```diff
@@ -296,10 +322,11 @@ sub cleanup
 	my $mirror = shift @mirrors;
 	$mirror or die "No more mirrors to try - giving up.\n";
 
-	download($mirror, $url_filename);
+	download($mirror, $url_filename, @additional_mirrors);
 	if (!-f "$target/$filename" && $url_filename ne $filename) {
-		download($mirror, $filename);
+		download($mirror, $filename, @additional_mirrors);
```

Comment:
This and the other cause aria2 not working. Pls revert in this function, additional_mirrors to mirrors.

With that fixed, I will merge this :D